docker/Dockerfile_ros-melodic

```diff
@@ -31,6 +32,9 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key C1CF6E31E6B
 		ros-$ROS_DISTRO-rostest \
 		ros-$ROS_DISTRO-rosunit \
 		ros-$ROS_DISTRO-xacro \
+		ros-$ROS_DISTRO-octomap \
```

Comment:
Please order this alphabetically with the other packages.

Reply:
Done!